### PR TITLE
Ovsdb metrics struct

### PR DESCRIPTION
**Pull request by safchain**

### CODE REVIEW

File: sflow/agent.go [hunk 1]
```diff
@@ -103,207 +104,156 @@ func (sfa *Agent) feedFlowTable() {
...
-		currMetric.Last = now
-
-		sfa.Graph.Lock()
 
-		var totalMetric *SFMetric
-		if metric, err := sfa.Node.GetField("SFlow.Metric"); err == nil {
-			prevMetric := metric.(*SFMetric)
-			currMetric.Start = prevMetric.Last
+		lastUpdateMetric := sf.Metric.Sub(prevMetric).(*SFMetric)
+		if lastUpdateMetric != nil && !lastUpdateMetric.IsZero() {
```
Comment (lebauce) [inline sflow/agent.go]: is `lastUpdateMetric != nil` really useful ?

File: sflow/sflow.go [hunk 2]
```diff
@@ -0,0 +1,595 @@
...
+
+//SFlow all sflow information
+// easyjson:json
+type SFlow struct {
+	IfMetrics        map[int64]*IfMetric `json:"IfMetric,omitempty"`
+	Metric           *SFMetric           `json:"Metric,omitempty"`
+	LastUpdateMetric *SFMetric           `json:"LastUpdateMetric,omitempty"`
+}
+
+//SFMetadataDecoder implements a json message raw decoder
```
Comment (lebauce) [inline sflow/sflow.go]: nit: space after //

File: sflow/sflow.go [hunk 3]
```diff
@@ -0,0 +1,595 @@
...
+
+import (
+	"encoding/json"
+	"strings"
+
+	"github.com/skydive-project/skydive/common"
+	"github.com/skydive-project/skydive/topology"
+)
+
+//SFlow all sflow information
```
Comment (lebauce) [inline sflow/sflow.go]: nit: space after //

**Comment by safchain** [comment]
rerun skydive-functional-tests-backend-orientdb

**Comment by safchain** [comment]
run skydive-scale-tests

**Comment by safchain** [comment]
run skydive-functional-tests-backend-orientdb

**Pull request merged**